Sources/swift-algorand-sdk/v2/client/algod/SwaggerJson.swift

```diff
@@ -8,6 +8,8 @@
 
 import Foundation
 import Alamofire
+
+// TODO: is this really needed? Swagger in a Swift application?
```

Comment:
@Jesulonimi21 is Swagger response something really needed by a swift sdk? Just want to make sure this is not here by mistake

Reply:
Yeah, this should be left here, it can be used to get the current swagger spec, this is worth noting that the swift SDK can also be used on the server too